docs/guides/handle-links.php

```diff
@@ -29,7 +31,7 @@
     )]
     #[Get('/company/{companyId}/employees/{id}')]
     #[ORM\Entity]
-    class Employee
+    class Employee implements LinksHandlerInterface
```

Comment:
A callable is already used here using `stateOptions: new Options(handleLinks: [Employee::class, 'handleLinks'])`